src/clj/rems/api/schema.clj

```diff
@@ -148,6 +148,7 @@
    (s/optional-key :field/options) [{:key s/Str
                                      :label LocalizedString}]
    (s/optional-key :field/max-length) (s/maybe (s/constrained s/Int not-neg?))
+   (s/optional-key :field/privacy) (s/enum :public :private)
```

Comment:
schema should document that this defaults to public, right?

Reply:
It's part of the opaque JSON blob so I guess not?

Reply:
Ok done now with description.